app/views/admin/edition_translations/edit.html.erb

```diff
@@ -78,7 +78,12 @@
         {
           id: "history_tab",
           label: "History",
-          content: render(Admin::Editions::DocumentHistoryTabComponent.new(edition: @edition, document_history: @document_history, editing: true))
+          content: render(Admin::Editions::DocumentHistoryTabComponent.new(
+            edition: @edition,
+            document_history: @document_history,
+            editing: true,
+            pagination: paginate(@document_history, theme: 'history', param_name: 'history_page'),
```

Comment:
Is there a reason why you chose to pass the "rendered" pagination into the component?

This looks like the sort of thing we'd be able to do from within the component itself by using the `paginate` helper directly.

Reply:
As discussed calling it within the view component causes a bunch of issues. We're going pass it in and feature test it.